### PR TITLE
Update Secure AI Lifecycle docs and samples to replace SCZ / SantaCruz / etc terms

### PR DESCRIPTION
**Pull request by JiaBaoxi**
1. remove SCZ / SantaCruz / etc terms from docs
2. add "todo" items in the docs to track the left work
3. leave sczpy APIs unmodified

### CODE REVIEW

File: Sample-Scripts-and-Notebooks/Official/Secured Locker/azure-functions/README.md [hunk 1]
```diff
@@ -15,19 +15,19 @@ To run the sample, you need:
...
 "AZURE_CLIENT_SECRET": "",
 "AZURE_TENANT_ID": "",
-"scz_mm_server_url": "",
-"scz_mm_storage_account": "",
-"scz_mm_telemtry_storage_container": "data",
-"scz_mm_model_name": "",
-"scz_mm_model_version": "",
+"mm_server_url": "",
+"mm_storage_account": "",
+"mm_telemtry_storage_container": "data",
```
Comment (kalpesh94) [inline Sample-Scripts-and-Notebooks/Official/Secured Locker/azure-functions/README.md]: telemtry is missing an e, it should be tel**e**metry

File: README.md [hunk 2]
```diff
@@ -17,12 +17,13 @@ On this page you will find everything you need to get started using your dev kit
 - [Build a custom solution using Advanced code (VS Code + AML)](https://github.com/microsoft/Project-Santa-Cruz-Private-Preview/blob/main/Sample-Scripts-and-Notebooks/Official/MobileNetV2SSDL_TrainingonVSCodeIDE/readme.md)
 
 ## 4. Secure Your Model and Data
```
Comment (kalpesh94) [inline README.md]: Instead of Secure, let's use the term Protect

File: README.md [hunk 3]
```diff
@@ -17,12 +17,13 @@ On this page you will find everything you need to get started using your dev kit
...
-
-* [Setup a secured AI model locker](user-guides/secured_locker/deployment.md)
```
Comment (kalpesh94) [inline README.md:21]: delete secured

File: Sample-Scripts-and-Notebooks/Official/Secured Locker/azure-functions/README.md (hunk 1 above)
Comment (BaoxiJia) [reply Sample-Scripts-and-Notebooks/Official/Secured Locker/azure-functions/README.md]: Thanks. Fixed in the new revision.

File: README.md (hunk 2 above)
Comment (BaoxiJia) [reply README.md]: Thanks. Updated in the new revision.

File: README.md (hunk 3 above)
Comment (BaoxiJia) [reply README.md:21]: Thanks. Removed secured in the new revision.

**Pull request merged**